### PR TITLE
sync of framework 

### PR DESCRIPTION
**Pull request by dapparu**
- modifications of functions: cross-talk inversion, cluster cleaning, saturation correction
- add of Pixels SF
- add of Ias_StripOnly
- add and modification of config files

### CODE REVIEW

File: Analyzer/plugins/Analyzer.h [hunk 1]
```diff
@@ -295,7 +295,7 @@ class Analyzer : public edm::EDAnalyzer {
   double GlobalMaxEta = 2.1;      // cut on inner tracker track eta
   double GlobalMaxV3D = 99999;    //0.50 cuts away signal;   // cut on 3D distance (cm) to closest vertex
   double GlobalMaxDZ = 0.50;      // cut on 1D distance (cm) to closest vertex in "Z" direction
-  double GlobalMaxDXY = 0.50;     // cut on 2D distance (cm) to closest vertex in "R" direction
+  double GlobalMaxDXY = 0.50;     // cut on 2D distance (cm) to closest vertex in "R" direction //proposal: 0.05
```
Comment (tvami) [inline Analyzer/plugins/Analyzer.h:298]: and I think these should be configurable in the config files, but I can do this

File: Analyzer/interface/CommonFunction.h [hunk 2]
```diff
@@ -1232,14 +1646,16 @@ reco::DeDxData computedEdx(const reco::DeDxHitInfo* dedxHits,
     }
 
     double scaleFactor = scaleFactors[0];
-    if (detid.subdetId() < 3)
+    if (detid.subdetId() < 3){
       scaleFactor *= scaleFactors[1];  // add pixel scaling
+      double pixelScaling = GetSFPixel(detid.subdetId(), detid, year, iEvent.id().run());
```
[the remark this answers is not in the document]
Comment (enibigir) [reply Analyzer/interface/CommonFunction.h:1651]: I think so.
just wanted to ask if on top of being  float-s, can we also use scaleFactors as a vector?

File: Analyzer/test/HSCParticleProducerAnalyzer_cfg.py [hunk 3]
```diff
@@ -101,27 +106,24 @@
...
-      ]
-   elif(options.SAMPLE=='isBckg'):
-      #to be updated to Run2 Triggers, in the meanwhile keep all of them to study trigger efficiency
-      process.HSCPTrigger.HLTPaths = [
-          "HLT_PFMET120_PFMHT120_IDTight_v*", #check triggers  
-          "HLT_Mu50_v*",
-          "HLT_PFHT500_PFMET100_PFMHT100_IDTight_v*",
-          "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_PFHT60_v*",
-          "HLT_MET105_IsoTrk50_v*",
+          "HLT_OldMu100_*",
```
Comment (tvami) [inline Analyzer/test/HSCParticleProducerAnalyzer_cfg.py:109]: Are we adding this back?

Comment (dapparu) [reply Analyzer/test/HSCParticleProducerAnalyzer_cfg.py:109]: I think so...

**Pull request merged**